style2.css

```diff
@@ -0,0 +1,49 @@
+.burger {
+	background-color: salmon;
+	display: none;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5813015)